test/cli-test.js

```diff
@@ -0,0 +1,196 @@
+//Sample test
+import assert from 'assert';
+import fs from 'fs';
+import fsExtra from 'fs-extra';
+
+// Tests need to import transpiled files that will be located in dist/ rather than src/
+var cli = require('../dist/cli');
+import constants from '../dist/constants/constants';
+
+const configs = {
+  projectName: 'jack',
+  description: 'Jack\'s Test Project',
+  version: '0.0.1',
+  author: 'Jack Meyer',
+  license: 'MIT',
+  ci: 'TravisCI',
+  container: 'Docker'
+};
+
+describe('Init Project', () => {
+  describe('#initProject()', () => {
```

Comment:
You could avoid having excessive describe blocks

Reply:
I've changed this.